control/include/http/check_data.h

```diff
@@ -77,9 +77,9 @@ class CheckData {
                           std::string *value) const = 0;
 
   // Extracts authentication information from header. Returns true on success,
-  // and saves authentication information in |attrs|. Returns false otherwise.
-  virtual bool GetAuthenticationHeader(
-      std::map<std::string, std::string> *attrs) const = 0;
+  // and saves authentication information in |payload|. Returns false otherwise.
```

Comment:
Please update the comment as
// If the request has a JWT token and it is verified, get its payload as string map.

Reply:
Done. Please take a look. Thanks.